AppCenter/AppCenter/Internals/HttpClient/MSHttpClientProtocol.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+typedef void (^MSHttpRequestCompletionHandler)(NSHTTPURLResponse *response, NSData *data, NSError *error);
+
+@protocol MSHttpClientProtocol <NSObject>
```

Comment:
Do you need `<NSObject>`?

Reply:
Not strictly speaking but some people recommend doing this either way. I don't see a convention in our code. I'll remove it because I don't feel strongly either way.